<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="T10-ACA-0599" failures="1" tests="1" name="Surefire test" time="10.551" errors="0" timestamp="30 Jan 2020 08:55:54 GMT">
  <testcase classname="com.frameworkium.bdd.TestRunner" name="test" time="0.865">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.cttest.stepDefs.login.theUserIsLoggedIn(login.java:13)
at âœ½.the customer user is logged in(file:src/test/resources/features/basket.feature:5)
]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- Surefire test -->
